Sources/SwiftSyntax/generated/SyntaxCollections.swift

```diff
@@ -124,22 +125,18 @@ public struct AttributeListSyntax: SyntaxCollection, SyntaxHashable {
       self = .ifConfigDecl(node)
     }
     
-    public init?(_ node: some SyntaxProtocol) {
-      if let node = node.as(AttributeSyntax.self) {
+    public init?(_ node: __shared some SyntaxProtocol) {
+      if let node = AttributeSyntax(node) {
```

Comment:
I personally prefer the `node.as(AttributeSyntax.self)` casting style because it’s closer to standard Swift.

Reply:
I just blindly copied that from the non-raw side ;)